src/shaderlib/picking/picking.js

```diff
@@ -0,0 +1,95 @@
+const DEFAULT_MODULE_OPTIONS = {
+  pickingSelectedColor: null, //  Set to a picking color to visually highlight that item
+  pickingThreshold: 1.0,
+  pickingActive: false // Set to true when rendering to off-screen "picking" buffer
+};
```

Comment:
**REQUIRED CHANGE** So I think the idea is that on the first run, `getUniforms` is called with the DEFAULT_MODULE_OPTIONS. After you removed it, we no longer have a `highlightColor` in the defaults, so this uniform will be undefined, at least in luma.gl.